docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to the Rust Exercism Track
+
+This document is a work-in-progress.
+Current maintainers are primarily preparing for the v3 launch of the Rust track.
```

Comment:
This line will get stale fast. One way we might future-proof it is by phrasing the paragraph like:

This track is a work in progress. Please take all the value you can from it, but if you notice any way to improve it, we are eager for pull requests. Fixing a typo in documentation is every bit as valid a contribution as a massive addition of code.

> A work of art is never finished, merely abandoned.
>
> -- Paul Valéry

Reply:
I love the Paul Valéry quote. Frames our work quite wonderfully.